Cargo.toml

```diff
@@ -54,11 +54,13 @@ rtc = ["rtcc"]
 rt = ["stm32l0/rt"]
 disable-linker-script = []
 
+device-selected = []
```

Comment:
Maybe this should be renamed to `family-selected`? The three `stm32l0x[123]` features shouldn't really be used beyond PAC feature preselection, because they don't really say much about the features of that family. See https://github.com/stm32-rs/stm32l0xx-hal#mcu-features-mcu-.

An alternative would be to get rid of those features entirely, and to depend on `stm32l0/xxx` directly from the generated `mcu-*` features.

Reply:
> An alternative would be to get rid of those features entirely, and to depend on `stm32l0/xxx` directly from the generated `mcu-*` features. What do you think, @hannobraun? (Would be out of scope for this PR though.)

I don't know whether those are used somewhere else, but sure, if it's possible, I'm all for it.